app/helpers/registration_helper.rb

```diff
@@ -0,0 +1,7 @@
+module RegistrationHelper
+  def show_phone_field?
+    return true unless Rails.application.config.feature_flag_enforce_levels_of_authentication
+
+    params.dig(:authenticate_to_level) == "level0"
```

Comment:
This is the wrong way around: we want to show the phone field if they're signing up to `level1`

Reply:
yep just amended that as the login_specs were angry.
force push shortly!